appveyor.yml

```diff
@@ -0,0 +1,35 @@
+
+os: Visual Studio 2015
+
+environment:
+  matrix:
+    - ARCH:                x64
+      D_COMPILER:          dmd
+      D_VERSION:           2.077.1
+      C_COMPILER:          MSVC
+
+artifacts:
+  - path: src/dmd.exe
```

Comment:
Not sure if this should be removed or not, it just allows for the win64 executable to be easily accessible. Can just replace the 32-bit one, after it is installed with this.

Reply:
Why would you want to provide artifacts?
We already have http://nightlies.dlang.org/ for nightlies.

Reply:
> We already have http://nightlies.dlang.org/ for nightlies.

It doesn't provide a 64-bit Windows executable of dmd, does it?